src/util/Constants.js

```diff
@@ -371,6 +371,12 @@ exports.DefaultAvatars = {
   RED: '1cbd08c76f8af6dddce02c5138971129',
 };
 
+exports.UserFlags = {
```

Comment:
Assuming this represents a bitfield, the keys/vals should probably be swapped around